nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -113,9 +114,11 @@ def broadcast_model(
             task_name (str, optional): name of the task. Defaults to "train".
             data (FLModel, optional): FLModel to be sent to clients. If no data is given, send empty FLModel.
             targets (List[str], optional): the list of target client names or None (all clients). Defaults to None.
+            min_responses (int, optional): the minimum number of responses expected. If == 0, must receive responses from
```

Comment:
This was how the original internal controller uses min_responses https://github.com/NVIDIA/NVFlare/blob/main/nvflare/apis/impl/bcast_manager.py#L69

I can change ModelController to default to None to be more clear, but we should probably keep it internally to still use 0 for backwards compatibility for old controllers. Also min_response=0 wouldn't make sense anyways, as it would automatically end the task.